Update Ibexa Cloud CLI (#2220)
ibexa/documentation-developer: docs/getting_started/install_on_ibexa_cloud.md

git push -u <platform.sh-remote> master
```

You can also use the [Ibexa Cloud CLI](https://cli.ibexa.co/) to push your code.

``` bash
ibexa_cloud push master
```

!!! note

    `master` is the Platform.sh name for the production branch.

!!! caution

>     Do not use Platform.sh CLI (`platform`) but [Ibexa Cloud CLI (`ibexa_cloud`)](https://cli.ibexa.co/).

Reviewer: ```suggestion
    Do not use Platform.sh CLI (`platform`), instead, use the [Ibexa Cloud CLI (`ibexa_cloud`)](https://cli.ibexa.co/).
```

    To install Ibexa Cloud CLI, follow https://cli.ibexa.co/ "Installation instructions".

>     Ibexa Cloud CLI and Platform.sh CLI share the same commands and the [same documentation](https://docs.platform.sh/administration/cli.html#3-use) whre you just have to replace `platform` with `ibexa_cloud`.

Reviewer: ```suggestion
    Ibexa Cloud CLI and Platform.sh CLI share the same commands and the [same documentation](https://docs.platform.sh/administration/cli.html#3-use) where you have to replace `platform` with `ibexa_cloud`.
```

>     If you have previously set up an alias to use Platform.sh CLI with Ibexa Cloud, it is outdated. Please, remove the alias and install Ibexa Cloud CLI instead.

Reviewer: ```suggestion
    If you have previously set up an alias to use Platform.sh CLI with Ibexa Cloud, it is outdated. Remove the alias and install Ibexa Cloud CLI instead.
```

git push -u <platform.sh-remote> master
```

You can also use the [Ibexa Cloud CLI](https://cli.ibexa.co/) to push your code.

``` bash
ibexa_cloud push master
```

!!! note

    `master` is the Platform.sh name for the production branch.

!!! caution

    Do not use Platform.sh CLI (`platform`), instead, use the [Ibexa Cloud CLI (`ibexa_cloud`)](https://cli.ibexa.co/).

    To install Ibexa Cloud CLI, follow https://cli.ibexa.co/ "Installation instructions".

>     Ibexa Cloud CLI and Platform.sh CLI share the same commands and the [same documentation](https://docs.platform.sh/administration/cli.html#3-use) where you have to replace `platform` with `ibexa_cloud`.

Reviewer 2: ```suggestion
    Ibexa Cloud CLI and Platform.sh CLI share the same commands and the [same documentation](https://docs.platform.sh/administration/cli.html#3-use) but you have to replace `platform` with `ibexa_cloud`.
```

Is it what is meant here? Tha
```suggestion
    Ibexa Cloud CLI and Platform.sh CLI share the same commands and the [same documentation](https://docs.platform.sh/administration/cli.html#3-use), but you have to replace `platform` with `ibexa_cloud`.
```

Is it what is meant here? That you have to replace `platform` with `ibexa_cloud` in commands and docs?

Author: Yes, this is still the right meaning.
Use the same doc but replace `platform` with `ibexa_cloud`.
